documentation/staging/content/security/domain-security/weblogic-channels.md

```diff
@@ -13,9 +13,62 @@ unless absolutely necessary. If exposing any of these channels, limit access usi
 controls like security lists or set up a Bastion to provide access.
 {{% /notice %}}
 
-When accessing T3 or RMI based channels, the preferred approach is to `kubectl exec` into
-the Kubernetes Pod and then run `wlst`, or set up Bastion access and then run
-`wlst` from the Bastion host to connect to the Kubernetes cluster.
+When accessing T3 or RMI based channels for administrative purposes
+such as running WLST, the preferred approach is to `kubectl exec` into
+the Kubernetes Pod and then run `wlst.sh`, or set up Bastion access and then run
+`java weblogic.WLST` or `$ORACLE_HOME/oracle_common/common/bin/wlst.sh`
+from the Bastion host to connect to the Kubernetes cluster
+(some cloud environments use the term Jump Host or Jump Server instead of Bastion).
 
 Also, consider a private VPN if you need use cross-domain T3 access
 between clouds, data centers, and such.
+
+#### WebLogic HTTP channels
+
+When providing remote access to HTTP using a load balancer or similar,
+Oracle recommends relaying the traffic to a dedicated
+WebLogic Server port that you have configured
+using a custom HTTP channel (network access point)
+instead of relaying the traffic to a default port.
+This helps ensure that external
+traffic is limited to the HTTP protocol. A custom HTTP channel
+is preferred over a default port because a default port supports
+multiple protocols.
+
+Do not enable tunneling on an HTTP channel unless you specifically
```

Comment:
One of the new security validations complains if tunneling is simply enabled. Given that, we probably should more clearly recommend against using tunneling unless the port is more protected on the network.

Reply:
Done.